packages/components/psammead-social-embed/src/Notice/index.jsx

```diff
@@ -57,14 +57,22 @@ const Notice = ({
   warningText,
 }) => {
   const dictionary = dictionaryFactory({ provider });
+  const detokenisedLinkText = detokenise(linkText, dictionary);
+
   return (
     <Wrapper service={service}>
       <p>{detokenise(text, dictionary)}</p>
-      <a href={linkHref}>
-        {detokenise(linkText, dictionary)}
-        {linkTextSuffixVisuallyHidden && (
-          <span>{detokenise(linkTextSuffixVisuallyHidden, dictionary)}</span>
-        )}
+      <a
+        href={linkHref}
+        aria-label={
+          linkTextSuffixVisuallyHidden &&
+          `${detokenisedLinkText}${detokenise(
```

Comment:
is it worth pulling this out to its own variable like `detokenisedLinkText ` on L60 ? I think it would improve the readability a bit.

Reply:
Yes, good suggestion!

Reply:
`detokenise(text, dictionary)` expects `text` to be a string, and because `linkTextSuffixVisuallyHidden` is an optional prop, using a local variable might result in less readable code. Which do you prefer?

```js
const detokenisedLinkTextSuffix =
  linkTextSuffixVisuallyHidden &&
  detokenise(linkTextSuffixVisuallyHidden, dictionary);
// ...
aria-label={
  detokenisedLinkTextSuffix &&
  `${detokenisedLinkText}${detokenisedLinkTextSuffix}`
}
```

```js
aria-label={
  linkTextSuffixVisuallyHidden &&
  `${detokenisedLinkText}${detokenise(
    linkTextSuffixVisuallyHidden,
    dictionary,
  )}`
}
```

Reply:
Personally I prefer Option 1 :D. However this is not about preference, what you have works and works well! Let's merge this.